Docs inclusive language

christinaausley opened the pull request:
## Description

Closes https://github.com/camunda/camunda-platform-docs/issues/2453.

## To answer (@akeller thoughts?)

### See

We progressively need to eliminate usage of "see X link" and replace it with "refer to X link", but there are still about 400 existing references in the `next` version of docs. You may note I adjusted a handful of these already, but is it worth adjusting all, or cleaning them up as we move forward? Let me know the best use of time here 👍 

### IP Whitelist

Ideally, we wouldn't use "whitelist", but we refer to IP-Whitelist in our docs. Would this be an exception, or should I go ahead and adjust these references as well?

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

akeller reviewed (commented):
This is huge!! But important! Thanks for your work here.

Regarding links, let's take a best effort approach. As you see them or review new PRs, fix them (and advocate internally for folks opening PRs to be on the look out - don't use "see"!). I don't think we need to fix all of them immediately. 

For the IP Whitelist topic, we have an [open epic](https://github.com/camunda/product-hub/issues/1360) with the product, since it would need to change there first. In the epic, we should leave a comment that this is an exception to inclusive language in our style guide and ask that it be prioritized.

akeller commented on docs/apis-tools/community-clients/micronaut.md:
```diff
@@ -7,7 +7,7 @@ The Micronaut integration is a community extension allowing you to leverage Zeeb
 The integration provides a wrapper around the [Zeebe Java Client](/apis-tools/java-client/index.md), and is preconfigured with sensible defaults so you can get started with minimal configuration. Add a dependency, implement a worker, and add your credentials in your Micronaut project.
 
-The Micronaut Framework is known for its efficient use of resources. Native images created with [GraalVM](https://www.graalvm.org/) reduce startup times to milliseconds.
+The Micronaut Framework is known for its efficient use of resources. Built-in images created with [GraalVM](https://www.graalvm.org/) reduce startup times to milliseconds.
```
I was just discussing this on the DevRel Collective Slack. I don't know if this is the best substitution here, but I can't think of a better one. In the context of images this is still a well-known term. Could you do an internal poll in ask-documentation or an external community poll and get more feedback on this?

christinaausley replied:
You may find [this page](https://www.graalvm.org/22.0/reference-manual/native-image/) interesting given the context here.

As this is used in code snippets and defined by GraalVM, this may have to remain an exception.

christinaausley replied:
Noted in the Confluence page as an exception 👍

akeller commented on docs/components/best-practices/development/testing-process-definitions.md:
```diff
@@ -168,7 +168,7 @@ void testDuplicate() throws Exception {
 ### Drive the process and assert the state
 
-For tests, you drive the process from waitstate to waitstate and assert that you see the expected process and variable states. For example, you might implement a test to test the scneario that a tweet gets approved:
+For tests, you drive the process from waitstate to waitstate and assert that you observe the expected process and variable states. For example, you might implement a test to test the scenario that a tweet gets approved:
```
Wow! Great spelling error catch!

christinaausley replied:
Figured why not fix it along the way!

christinaausley commented:
In the latest commit, I've:

- Replaced terms for "native images" as an exception.
- Adjusted terms referencing "abort"

I've also noted all of these adjustments in Confluence, and will give the glossary a shoutout at the next WCoE release (next week!)

Just looking for final approval 👍

akeller reviewed (approved):
All of this sounds great! Can you also add it to our Sept internal release presentation list? All the shoutouts everywhere 🚀
